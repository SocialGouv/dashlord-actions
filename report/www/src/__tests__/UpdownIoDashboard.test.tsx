import { render } from "@testing-library/react";
import UpdownIo from "../../pages/updownio";

it("Should render UpdownIo", () => {
  const { container } = render(<UpdownIo />);
  expect(container).toMatchSnapshot();
});
