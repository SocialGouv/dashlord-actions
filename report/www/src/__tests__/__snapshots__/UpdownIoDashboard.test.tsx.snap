// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Should render UpdownIo 1`] = `
<div>
  <h1>
    Disponibilité et temps de réponse
  </h1>
  Consulter le tableau de bord updown.io :
   
  <a
    class="fr-link"
    href="https://updown.io/p/8c6x8"
    target="_blank"
  >
    https://updown.io/p/8c6x8
  </a>
  <br />
  <br />
  <div
    class="fr-callout"
    id="fr-callout-:r0:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/api-gouv-fr"
      >
        api.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            666ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r1:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              29/03/2024
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/z6jn"
      id="fr-button-:r2:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r3:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/api-trackdechets-beta-gouv-fr"
      >
        api.trackdechets.beta.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-alert fr-alert--info fr-mt-3w"
        id="fr-alert-:r4:"
      >
        <h3
          class="fr-alert__title"
        >
          Données absentes
        </h3>
        <p>
          Aucune donnée updown.io associée
        </p>
      </div>
       
    </p>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r5:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/annuaire-entreprises-data-gouv-fr"
      >
        annuaire-entreprises.data.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            578ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r6:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              06/04/2024
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/futp"
      id="fr-button-:r7:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r8:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/acceslibre-beta-gouv-fr"
      >
        acceslibre.beta.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            0.995
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            607ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r9:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              10/03/2024
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/by0u"
      id="fr-button-:ra:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:rb:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/aidantsconnect-beta-gouv-fr"
      >
        aidantsconnect.beta.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            0.91
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            839ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:rc:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              07/03/2024
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/tnjw"
      id="fr-button-:rd:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:re:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/archifiltre-fabrique-social-gouv-fr"
      >
        archifiltre.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            65ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:rf:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              05/08/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/egg2"
      id="fr-button-:rg:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:rh:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/index-egapro-travail-gouv-fr"
      >
        index-egapro.travail.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            53ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:ri:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              01/08/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/5o23"
      id="fr-button-:rj:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:rk:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/domifa-fabrique-social-gouv-fr"
      >
        domifa.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            99.818%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            66ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:rl:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              01/08/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/t6x4"
      id="fr-button-:rm:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:rn:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/www-fabrique-social-gouv-fr"
      >
        www.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            99.843%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            71ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:ro:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              18/07/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/rt57"
      id="fr-button-:rp:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:rq:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/pjm-social-gouv-fr"
      >
        pjm.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            62ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:rr:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              21/06/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/bv33"
      id="fr-button-:rs:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:rt:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/tumeplay-fabrique-social-gouv-fr"
      >
        tumeplay.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            99.993%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            135ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:ru:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              21/06/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/yp6o"
      id="fr-button-:rv:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r10:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/1000jours-fabrique-social-gouv-fr"
      >
        1000jours.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            72ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r11:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              04/08/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/3g81"
      id="fr-button-:r12:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r13:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/1000jours-blues-fabrique-social-gouv-fr"
      >
        1000jours-blues.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            80ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r14:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              11/08/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/k3d1"
      id="fr-button-:r15:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r16:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/monpsy-sante-gouv-fr"
      >
        monpsy.sante.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            100%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            68ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r17:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              18/07/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/na4t"
      id="fr-button-:r18:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r19:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/ressourcerie-fabrique-social-gouv-fr"
      >
        ressourcerie.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            99.767%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            71ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r1a:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              10/08/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/ko71"
      id="fr-button-:r1b:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r1c:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/admin-domifa-fabrique-social-gouv-fr"
      >
        admin-domifa.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-alert fr-alert--info fr-mt-3w"
        id="fr-alert-:r1d:"
      >
        <h3
          class="fr-alert__title"
        >
          Données absentes
        </h3>
        <p>
          Aucune donnée updown.io associée
        </p>
      </div>
       
    </p>
  </div>
  <div
    class="fr-callout"
    id="fr-callout-:r1e:"
  >
    <h3
      class="fr-callout__title"
    >
      <a
        href="/url/template-fabrique-social-gouv-fr"
      >
        template.fabrique.social.gouv.fr
      </a>
    </h3>
    <p
      class="fr-callout__text"
    >
       
      <div
        class="fr-grid-row fr-grid-row--gutters fr-mt-3w"
      >
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Taux de disponibilité sur un mois glissant
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            99.915%
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            APDEX: Application Performance Index
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            1
          </div>
        </div>
        <div
          class="_metric_dd33f8 fr-col-4"
        >
          <div />
          <div
            class="_metricTitle_dd33f8"
          >
            Temps de réponse
          </div>
          <div
            class="_metricValue_dd33f8"
          >
            71ms
          </div>
        </div>
      </div>
      <div
        class="_left_dd33f8"
      >
        <div
          class="fr-mb-3w"
        >
          <div
            style="padding: 10px;"
          >
            <br />
            <br />
            Le certificat SSL est
             
            <p
              class="fr-badge fr-badge--success fr-badge--no-icon"
              id="fr-badge-:r1f:"
              style="font-size: 1em; width: 100px; text-align: center; display: inline-block;"
            >
              valide
            </p>
             
            et expire le
             
            <strong>
              27/06/2022
            </strong>
          </div>
        </div>
      </div>
       
    </p>
    <a
      class="fr-btn"
      href="https://updown.io/4h15"
      id="fr-button-:r1g:"
      target="_blank"
    >
      Statistiques détaillées
    </a>
  </div>
</div>
`;
